webapp/channels/src/components/admin_console/ldap_wizard/ldap_wizard.tsx

```diff
@@ -547,27 +635,59 @@ const LDAPWizard = (props: Props) => {
 
         return (
             <div>
-                {sections}
+                {renderedSections}
             </div>
         );
     };
 
     return (
         <div
-            className={'wrapper--fixed'}
+            className={'wrapper--fixed ldap-wizard-wrapper'}
             data-testid={`sysconsole_section_${schema?.id}`}
         >
             {renderTitle()}
-            <div className='admin-console__wrapper'>
-                <div className='admin-console__content'>
-                    <form
-                        className='form-horizontal'
-                        role='form'
-                        onSubmit={handleSubmit}
-                    >
-                        {renderSettings()}
-                    </form>
-                    {hybridSchemaAndComponent()}
+            <div className='ldap-wizard-content-wrapper'>
```

Comment:
Could the sidebar be pulled out into a separate component to make the whole component smaller?

Reply:
Ah, I misunderstood. Yep, that makes sense -- I extracted it into a `renderSidebar()`. It's a clearer now, thanks!  @hmhealey